test/CronTest.cpp

```diff
@@ -446,3 +448,145 @@ SCENARIO("Tasks can be added and removed from the scheduler")
         }
     }
 }
+
+SCENARIO("Testing CRON-Tick Performance")
+{
+    GIVEN("A Cron instance with no task")
+    {
+        using clock = std::chrono::high_resolution_clock;
+        using std::chrono::milliseconds;
+        using std::chrono::duration_cast;
+
+        Cron<TestClock> c1{};
+        auto& cron_clock1 = c1.get_clock();
+        
+        Cron<TestClock> c2{};
+        auto& cron_clock2 = c2.get_clock();
+
+        Cron<TestClock> c3{};
+        auto& cron_clock3 = c3.get_clock();
+        
+        int count1 = 0;
+        int count2 = 0;
+        int count3 = 0;
+
+        WHEN("Creating 1000 CronData Objects")
+        {
+            std::string cron_job = "* * * * * ?";
+            auto begin_cron_data = clock::now();
+            for(int i = 1; i <= 1000; i++)
+            {
+                auto cron = CronData::create(cron_job);
+            }
+            auto end_cron_data = clock::now();
+            auto msec_cron_data = duration_cast<milliseconds>(end_cron_data - begin_cron_data).count();
+            
+            // Hopefully executing a more or less empty task does only take some milliseconds
+            REQUIRE(msec_cron_data <= 1000);
```

Comment:
Use chrono-types to ensure type safety instead of free numbers?

Reply:
True!